Common/Data/Custom/PsychSignal/PsychSignalSentimentData.cs

```diff
@@ -0,0 +1,137 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using System;
+using System.Globalization;
+using System.IO;
+
+namespace QuantConnect.Data.Custom.PsychSignal
+{
+    public class PsychSignalSentimentData : BaseData
+    {
+        /// <summary>
+        /// Bullish intensity as reported by psychsignal
+        /// </summary>
+        public decimal BullIntensity;
```

Comment:
I think the prevailing style in the code base is to use properties:
``` csharp
public decimal BullIntensity { get; set; }
```

Reply:
Agreed, but I did this for another reason: The cloning of the object would not successfully clone members who were implemented as properties. I didn't know this before I implemented the class, but just commenting here in case someone ever stumbles across their custom data variables being null. The solution to that problem is to implement the clone method for the custom data source.